commons/typed-values/src/test/java/org/operaton/bpm/engine/test/api/variable/VariableApiTest.java

```diff
@@ -49,127 +48,127 @@ class VariableApiTest {
-    assertEquals(SerializationDataFormats.JSON.getName(), objectValue.getSerializationDataFormat());
+      objectValue = objectValue(DESERIALIZED_OBJECT_VAR_VALUE).serializationDataFormat(SerializationDataFormats.JSON).create();
+      assertThat(objectValue.getSerializationDataFormat()).isEqualTo(SerializationDataFormats.JSON.getName());
 
-    objectValue = objectValue(DESERIALIZED_OBJECT_VAR_VALUE).serializationDataFormat(SerializationDataFormats.XML).create();
-    assertEquals(SerializationDataFormats.XML.getName(), objectValue.getSerializationDataFormat());
+      objectValue = objectValue(DESERIALIZED_OBJECT_VAR_VALUE).serializationDataFormat(SerializationDataFormats.XML).create();
+      assertThat(objectValue.getSerializationDataFormat()).isEqualTo(SerializationDataFormats.XML.getName());
   }
 
   @Test
   void emptyVariableMapAsVariableContext() {
-    VariableContext varContext = createVariables().asVariableContext();
-    assertEquals(0, varContext.keySet().size());
-    assertNull(varContext.resolve("nonExisting"));
-    assertFalse(varContext.containsVariable("nonExisting"));
+      VariableContext varContext = createVariables().asVariableContext();
+      assertThat(varContext.keySet().size()).isEqualTo(0);
+      assertThat(varContext.resolve("nonExisting")).isNull();
+      assertThat(varContext.containsVariable("nonExisting")).isFalse();
   }
 
   @Test
   void testEmptyVariableContext() {
-    VariableContext varContext = emptyVariableContext();
-    assertEquals(0, varContext.keySet().size());
-    assertNull(varContext.resolve("nonExisting"));
-    assertFalse(varContext.containsVariable("nonExisting"));
+      VariableContext varContext = emptyVariableContext();
+      assertThat(varContext.keySet().size()).isEqualTo(0);
+      assertThat(varContext.resolve("nonExisting")).isNull();
+      assertThat(varContext.containsVariable("nonExisting")).isFalse();
   }
 
   @Test
   void variableMapAsVariableContext() {
-    VariableContext varContext = createVariables()
-        .putValueTyped("someValue", integerValue(1)).asVariableContext();
+      VariableContext varContext = createVariables()
+              .putValueTyped("someValue", integerValue(1)).asVariableContext();
 
-    assertEquals(1, varContext.keySet().size());
+      assertThat(varContext.keySet().size()).isEqualTo(1);
 
-    assertNull(varContext.resolve("nonExisting"));
-    assertFalse(varContext.containsVariable("nonExisting"));
+      assertThat(varContext.resolve("nonExisting")).isNull();
+      assertThat(varContext.containsVariable("nonExisting")).isFalse();
 
-    assertEquals(1, varContext.resolve("someValue").getValue());
-    assertTrue(varContext.containsVariable("someValue"));
+      assertThat(varContext.resolve("someValue").getValue()).isEqualTo(1);
+      assertThat(varContext.containsVariable("someValue")).isTrue();
   }
 
   @Test
-  void transientVariables() throws Exception {
+  void transientVariables() throws URISyntaxException {
```

Comment:
This has recently changed. Seems that you worked on the older state, might had conflicts and now have the old state again. `throws Exception` is correct.

Please clean this up by running
```
./mvnw -U org.openrewrite.maven:rewrite-maven-plugin:run -Drewrite.recipeArtifactCoordinates=org.openrewrite.recipe:rewrite-testing-frameworks:RELEASE -Drewrite.activeRecipes=org.openrewrite.java.testing.cleanup.SimplifyTestThrows -Drewrite.exportDatatables=true
```